### ISSUE TITLE
Long new flat page creation

### ISSUE THREAD

**Issue by duker33** [fidals/stroyprombeton#637]
Serv generates this page about 8s. Can't reproduce it with my local app
https://www.stroyprombeton.ru/admin/pages/flatpage/add/

**Comment by duker33** [comment]
every flat page has a parent input with all pages inside.
Checked DB and found out that we have many orphan model pages:
```
>>> from pages.models import ModelPage
>>> ModelPage.objects.filter(stroyprombeton_product=None).count()
14355
>>> ModelPage.objects.count()
15252
```

So, i'll rm them from the prod DB

**Comment by duker33** [comment]
seems we have some problems with lost products or lost images:
```
>>> pp = (
...     ModelPage.objects
...     .filter(related_model_name='stroyprombeton_product')
...     .filter(stroyprombeton_product=None)
...     .filter(stroyprombeton_series=None)
...     .filter(stroyprombeton_category=None)
... ).first()
>>> pp.images.all()
<ImageQuerySet [<Image: 1>]>
>>> p = Product.objects.get(name=pp.name)
>>> p.page.images.all()
<ImageQuerySet []>
```

**Comment by duker33** [comment]
seems we should relaunch all db scripts for the first to preserve lost images.
And then launch this script

**Comment by duker33** [comment]
fixed it by #643
